### DISCUSSION TITLE
updating website with npm run deploy

### DISCUSSION THREAD

**Discussion by parisasalehii** [community/community#135813] [npm]
We have a website for our lab, and I usually update it using the command npm run deploy. It used to work fine, but recently, even though the command runs correctly and says the changes are published, the website doesn't reflect the updates.

**Comment by ebndev** [comment]
Hi @parisasalehii, thanks for being a part of the GitHub Community!

Are you able to share your repo / the commit with the changes? That might help someone give you a nudge in the right direction as it may not be an issue with the command or build itself.

**Comment by parisasalehii** [comment]
<img width="885" alt="Screenshot 2024-08-14 at 2 03 31 PM" src="https://github.com/user-attachments/assets/8bc807c5-5700-468a-a9a0-b8029a8b861b">
